### DISCUSSION TITLE
Facing error in build ☹

### DISCUSSION THREAD

**Discussion by leamandeep** [vercel/next.js#39145] [Help]
Hi Team ✋ !! I'm facing difficulty in production mode . When i run npm run build it's showing me error . My code is running well without any error in development but i don't know why it's showing such error ☹. Please help me to fix this  🙏.

Here are few ss of my code

![image](https://user-images.githubusercontent.com/68345524/181580157-698a7ef3-8057-4630-a4c4-ae5d33e7fcd3.png)

![image](https://user-images.githubusercontent.com/68345524/181580530-1f80cc51-75a4-4495-8963-32c7bbe5b052.png)

**Comment by GabenGar** [comment]
It quite literally shows you the errors along with the offending files. Install [`eslint` VSCode extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and you will see these errors in development too.

**Reply by leamandeep** [reply]
But why is it showing this ?? 🤔

error - unhandledRejection: Error: aborted
at connResetException (node:internal/errors:691:14)
at abortIncoming (node:_http_server:596:17)
at socketOnClose (node:_http_server:590:3)
at Socket.emit (node:events:402:35)
at TCP. (node:net:687:12) {
code: 'ECONNRESET'
}

**Reply by leamandeep** [reply]
Still Not 

> It quite literally shows you the errors along with the offending files. Install [`eslint` VSCode extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and you will see these errors in development too.

Still Not working ☹ . It's showing same error.

![image](https://user-images.githubusercontent.com/68345524/181758592-4abf937e-498d-4fe6-972c-765dd94b3e94.png)

**Reply by GabenGar** [reply]
I didn't say it would fix your issues, just that you will see these errors in the development. At this point you have to read the error message and fix the problem it tells you to fix.

**Reply by TomSmedley** [reply]
I'm getting the exact same error. 

https://github.com/vercel/next.js/discussions/39242

**Reply by leamandeep** [reply]
> I'm getting the exact same error.
> 
> #39242

Wait for version 4. Maybe issue would fix
